src/licensedcode/data/licenses/clips.LICENSE

```diff
@@ -0,0 +1,26 @@
+---
+key: clips
```

Comment:
We have updated this to the old clips-2017 license but not sure if the owner was from SPDX, it might have been my doing, so in that case we do not need to file an issue on SPDX?